docs/README.md

```diff
@@ -3,35 +3,66 @@
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 This website template is also copied from [Flashbots Docs](https://github.com/flashbots/docs).
 
+## Prerequisites (Linux)
+
+- `Yarn` installation:
+
+```shell
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt update
+sudo apt install yarn
+```
+
+_Please note:_ `sudo apt install yarn` will also install nodejs, however, it may not be the version which you require.
+Therefore, you should use `nvm` to install the latest version of node.
+
+- `Nvm` installation:
+
+```shell
+  sudo apt-get update
+  sudo apt-get install build-essential checkinstall libssl-dev
+  curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.32.1/install.sh | bash
+```
+
+Currently, the latest lts version of node is `v16.15.1` called `Gallium`.
+
+```shell
+nvm install --lts=Gallium
+```
```

Comment:
`nvm install --lts` will install the latest LTS version automatically.
```suggestion
```shell
nvm install --lts
```
```